metricbeat/module/kubernetes/volume/data.go

```diff
@@ -75,6 +75,9 @@ func eventMapping(content []byte, logger *logp.Logger) ([]mapstr.M, error) {
 			if volume.Inodes > 0 {
 				kubernetes2.ShouldPut(volumeEvent, "fs.inodes.pct", float64(volume.InodesUsed)/float64(volume.Inodes), logger)
 			}
+			if volume.PvcRef.Name != "" {
+				kubernetes2.ShouldPut(volumeEvent, "persistentvolumeclaim.name", volume.PvcRef.Name, logger)
+			}
```

Comment:
I have the feeling that this way the field that will be generated will be `kubernetes.volume.persistentvolumeclaim.name` . Is that what you want?

Reply:
Wow, great static analysis :D.. Indeed, its creating the field you mention. I actually want to reuse the field `kubernetes.persistentvolumeclaim.name` to allow correlation, but I am honestly a little bit lost.

After some thinking I tried with:
```
kubernetes2.ShouldPut(volumeEvent, mb.ModuleDataKey+".persistentvolumeclaim.name", volume.PvcRef.Name, logger)
``` 

But it does not seem to work 🤔

Reply:
Ok, the line is correct now. For some reasons I were still using the old binary.

Reply:
I had the feeling 😂